docker-compose.yaml

```diff
@@ -1,22 +1,29 @@
+version: '3.8'
+
 services:
   mongodb:
     image: mongo:latest
     ports:
       - 27017:27017
     volumes:
       - mongodb-data:/data/db
-
+  
   redis-stack-server:
     image: redis/redis-stack-server:latest
     ports:
       - 6379:6379
     volumes:
-      - redis-data:/data/redis
+      - redis-data:/data/redis 
 
   talawa-api-container:
-    build: .
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
```

Comment:
We can't specify the Dev Docker file in this docker-compose as it will affect our current cloud instance deployments

Reply:
If there is only one Dockerfile we can remove the specifying of dockerfile. But here we have two docker files hence it necessary to specify which dockerfile it should should to build image

Reply:
Ye should have two docker-compose files too - `docker-compose.dev.yml` and `docker-compose.prod.yml`

Reply:
I am thinking about it .Ok let us use then two docker-compose files

Reply:
@vasujain275  I didn't modify much about docker-compose for it. Just introduced volumes and use of specific docker file to build image.